### PR TITLE
Test: To-Entities refactor to get explicit errors.

### PR DESCRIPTION
**Pull request by eloycoto**
Few changes in the to-entities test:

- Added a const to true/False to is easy to understand if has access or
not in validateConnectivity.
- Refactor `validateconnectivity` to instead of match true/false use the
custom `CMDSucess`, so errors will print command output.
- Move the manifestGet to  the context for entities, so with a simple
read you know what yaml is added.

Signed-off-by: Eloy Coto <eloy.coto@gmail.com>

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/5806)

### CODE REVIEW

**Comment by eloycoto** [comment]
test-me-please

File: test/k8sT/Policies.go [hunk 1]
```diff
@@ -162,21 +159,30 @@ var _ = Describe("K8sPolicyTest", func() {
 		}
 
 		validateConnectivity := func(expectWorldSuccess, expectClusterSuccess bool) {
+
+			// getMatcher return a helper.CMDSucess() matcher for sucess or fail situations.
```
Comment (ianvernon) [inline test/k8sT/Policies.go]: return --> returns

sucess --> success
fail --> failure

File: test/k8sT/Policies.go [hunk 2]
```diff
@@ -663,51 +669,68 @@ var _ = Describe("K8sPolicyTest", func() {
 			}
 		})
 
-		It("toEntities All", func() {
-			By("Installing toEntities All")
-			importPolicy(cnpToEntitiesAll)
+		Context("Validate to-entities policies", func() {
+			const (
+				WorldConnetivityDeny  = false
```
Comment (ianvernon) [inline test/k8sT/Policies.go]: typo in variable. Connetivity --> Connectivity. Fix this in all variables.

**Review by ianvernon — APPROVED** [review]

**Comment by eloycoto** [comment]
test-me-please

**Comment by ianvernon** [comment]
I am retracting approval - the tests still are skipped. @eloycoto is this needed for 1.3?

**Review by ianvernon — COMMENTED** [review]
Would like to see this included with the changes that re-enable the cluster / host entities tests, which are still skipped.

**Comment by eloycoto** [comment]
@ianvernon not all the test are skipped (Still two are in place), and in this way the debug of the issues will be much better, so another PR can enable the test. 

Happy to wait until enable other test, but I think that can be merged.

**Comment by ianvernon** [comment]
@eloycoto you are right, my mistake.

**Pull request merged**